personal_assistant/pa_tag/views.py

```diff
@@ -0,0 +1,24 @@
+from django.shortcuts import render, redirect
+from .models import Tag
+from .forms import TagForm
+
+def main(request):
+    tags =  Tag.objects.all()
+    context = {"tags":tags}
```

Comment:
Використовувати один стиль лапок:
```suggestion
    context = {'tags': tags}
```